### ISSUE TITLE
Obtain icon as 8-bits / grayscale image?

### ISSUE THREAD

**Issue by tinchodias** [OpenSmock/Iconography#7]
Hi @labordep is there some way in the icons database or in this project, to get the icons directly in grayscale format?

**Comment by labordep** [comment]
Hi @tinchodias you can get Ant Design icons, icons are in black color format by default (loaded from SVG files).

You need to download locally the ant design icon project (this is explains in the documentation but maybe not very clear, we need feedbacks ;-) ) and after you can use it with : 

`self antIconNamed: #search`

**Comment by labordep** [comment]
We are working on a capacity to dynamically download icon from an internet repository but this is not yet priority (we are working offline).

**Comment by tinchodias** [comment]
Yes I did read the README and downloaded the ant icon set to the local directory where the default setting expects it.
So, no complaints on the doc! We can close the issue.

When I asked I was a bit confused creating this demo, and finally it worked without that:
<img width="254" alt="Screenshot 2023-05-23 at 18 36 34" src="https://github.com/OpenSmock/Iconography/assets/3044265/e0121037-2d73-422b-8048-1d5e71139774">

Code:
```smalltalk
inputForm := self antIconNamed: #checkCircle.
inputSurface := AeCairoImageSurface fromForm: inputForm.

outputColors := Color wheel: 7.

"Create a surface and associated context"
outputSurface := AeCairoImageSurface
    extent: (inputSurface width * outputColors size) @ inputSurface height
    format: AeCairoSurfaceFormat argb32.
outputContext := outputSurface newContext.

"Paint the inputSurface with each color"
outputColors
    do: [ :eachColor |
        outputContext
            sourceColorRGBA: eachColor;
            maskSurface: inputSurface x: 0 y: 0 ]
    separatedBy: [
        outputContext
            translateByX: inputSurface width
            y: 0 ].

outputSurface asForm inspect
```

**Comment by tinchodias** [comment]
(Snippet uses Alexandrie: https://github.com/pharo-graphics/Alexandrie)
